### ISSUE TITLE
Close Icon gives a11y warning

### ISSUE THREAD

**Issue by mtliendo** [emmabostian/fem-design-systems#12]
When looking at the accessability tab for the modal stories, there is a a11y violation regarding a button.  Since the Primary,Secondary, and Tertiary buttons aren't throwing errors, my hunch is that it's the close icon--which is an SVG but behaves like a button.

![image](https://user-images.githubusercontent.com/5106417/75522273-39bcd380-59cf-11ea-85cd-eda0baac84e6.png)

Because this is made with styled components, I don't know the recommended fix (sorry!) 😅 

As far as I know, this could be intended as a discussion point in the workshop, but I just wanted to point it out and may have to just head to the workshop for the solution 😉
